app/views/container_image/show.html.haml

```diff
@@ -1,4 +1,10 @@
 - if @display == 'containers'
  = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
+- elsif 'container_nodes'.eql?(@display) && @showtype != "compare"
+ = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
+- elsif 'container_groups'.eql?(@display) && @showtype != "compare"
+ = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
+- elsif 'container_projects'.eql?(@display) && @showtype != "compare"
+ = render :partial => "layouts/gtl", :locals => {:action_url => "show/#{@record.id}"}
```

Comment:
@blomquisg actually, these will eventually be changed to use restful routes, so we won't need this logic in the view.